sdk/anomalydetector/ai-anomaly-detector-rest/src/anomalyDetectorRest.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getClient, ClientOptions } from "@azure-rest/core-client";
+import { KeyCredential } from "@azure/core-auth";
+import { AnomalyDetectorRestClient } from "./clientDefinitions";
+
+/**
+ * Initialize a new instance of the class AnomalyDetectorRestClient class.
+ * @param Endpoint type: string Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus2.api.cognitive.microsoft.com).
+ * @param ApiVersion type: string Anomaly Detector API version (for example, v1.1).
+ * @param credentials type: KeyCredential
+ */
+export default function createClient(
+  Endpoint: string,
+  ApiVersion: string,
+  credentials: KeyCredential,
+  options: ClientOptions = {}
+): AnomalyDetectorRestClient {
+  const baseUrl = options.baseUrl ?? `${Endpoint}/anomalydetector/${ApiVersion}`;
```

Comment:
It seems that code-gen has already handled the apiVersion issue, so I use the code-gen version of this file. 

Using this [link](https://github.com/saintzou66/azure-sdk-for-js/pull/2/files#diff-1aebd09d4f0df4e52c51dcbc6bfa057e8c5cda083c94bff5fff1ebb26ddc07c8) will cause duplication of apiVersion in request url like this:
<endpoint>/v1.1/v1.1/detect

@joheredi